linkerd/protocol/http/src/main/scala/io/buoyant/linkerd/protocol/HttpConfig.scala

```diff
@@ -85,9 +91,24 @@ case class HttpServerConfig(
   }
 }
 
+// Cribbed from https://gist.github.com/Aivean/6bb90e3942f3bf966608
+class HttpIdentifierConfigDeserializer extends JsonDeserializer[Option[Seq[HttpIdentifierConfig]]] {
+  override def deserialize(p: JsonParser, ctxt: DeserializationContext): Option[Seq[HttpIdentifierConfig]] = {
+    val codec = p.getCodec
+    codec.readTree[TreeNode](p) match {
+      case n: JsonNode if n.isArray =>
+        import scala.collection.JavaConversions._
```

Comment:
fwiw JavaConverters is the recommended replacement for JavaConversions.  Supports `n.asScala`